docs/api.md

```diff
@@ -169,7 +171,7 @@ https://kovan.interep.link/api/v1/groups/github/gold/538962495891655485574540269
 **GET** - Returns a Merkle tree proof.
 
 ```bash title="Shell"
-curl https://kovan.interep.link/api/v1/groups/github/gold/6014393454173820032764441533619576647480292883965697181546606218195926726207/proof
+curl https://kovan.interep.link/api/v1/groups/github/established/6014393454173820032764441533619576647480292883965697181546606218195926726207/proof
```

Comment:
were all these calls example working for real?
If yes we Will need to update the examples later

Reply:
I'm not sure but it's ok if they don't work, it's just an example